config/samples/kustomization.yaml

```diff
@@ -2,4 +2,5 @@
 resources:
 - datadog-agent-all.yaml
 - datadogmetric-v1alpha1.yaml
+- datadoghq_v1alpha1_datadogmonitor.yaml
```

Comment:
this is not named in the same format as the others (although it matches the filename); should i change it?

Reply:
talked to vincent about this, going to leave it as is since it does not matter too much.